internal/datastore/postgres/revisions.go

```diff
@@ -27,12 +30,17 @@ const (
 	//   %[2] Relationship tuple transaction table
 	//   %[3] Name of timestamp column
 	//   %[4] Quantization period (in nanoseconds)
+	//   %[5] Name of snapshot column
```

Comment:
`%[5]` isn't used in the query? did you mean for the column name to be parameterizable?